user_guide/accounts.rst

```diff
@@ -0,0 +1,287 @@
+
+
+.. _user-guide-accounts-create:
+
+Create Account from the UI
+--------------------------
+
+- Go to the *Sales → Accounts*
+
+- Click :guilabel:`Create Account` button
+
+- The *Create Account* :ref:`form <user-guide-ui-components-create-pages>` will appear:
+
+.. image:: ./img/accounts/accounts_create.png
+
+The following fields are mandatory and **must** be defined:
+
+.. csv-table:: Mandatory Entity Fields
+  :header: "Field", "Description"
+  :widths: 10, 30
+
+  "**Owner***","Limits the list of users that can manage the account to users, whose roles allow managing 
+  accounts assigned to the owner (e.g. the owner, members of the same business unit, system administrator, etc.).
+  
+  By default, the user creating the account is chosen."
+  "**Account Name***","The name used to refer to the account in the system."
+
+The rest of the fields are **optional**. They keep additional details about the account (such as its :term:`tags <Tag>`
+and related :term:`contacts <Contact>`) and may be left empty.
+
+If you need to record and process any other details of accounts, **custom fields** can be created. Their values will 
+be displayed in the *Additional* section.
+
+.. hint::
+
+    To create a custom field, go to *System → Entities → Entity Management → Lead* and click :guilabel:`Create Field`
+    button.
+  
+Once all the necessary fields have been defined, click the button in the right top corner of the page to save the 
+account in the system.
+
+.. hint::
+
+    You can also export and import accounts with :guilabel:`Export` and :guilabel:`Import` buttons as described in 
+    the :ref:`Export and Import Functionality <user-guide-import>` guide. 
+
+Accounts View Page
+------------------
+
+:ref:`View page <user-guide-ui-components-view-pages>` consists of several sections, namely:
+
+- **General**: general details of the account, such as its name, tags, description and all the contacts assigned for the
+  account. 
+
+.. image:: ./img/accounts/accounts_view_general.png
+
+.. note::
+
+    Once a :term:`customer identity <Customer Identity>` is assigned to an Account, all of its 
+    :term:`contacts <Contact>` are added for the account, however they can be removed and/other contacts can be added
+    from the :ref:`Edit form <user-guide-ui-components-create-pages>` of the account.
+
+- **Record Activities**: activities (calls, e-mails and tasks assigned to the account)
+
+.. image:: ./img/accounts/accounts_view_activities.png
+
+.. note::
+    
+    If an activity-related action was done for a customer or a contact, assigned to the account they will not be 
+    displayed. Only the activities performed directly for the account are available in the section.
+
+- **Additional Information**: details of any :term:`custom fields <Custom Field>` defined for the account.
+
+- **Sections with channel names**: each the section contain details of all the customers that are assigned to this 
```

Comment:
Lets explain multichannels in separate section. We should make it clear on how channels and customers are related to account.